pytest_plugins/rerun_rp/rerun_rp.py

```diff
@@ -83,25 +83,28 @@ def pytest_addoption(parser):
 
         Defect_types:
             Collect failed tests marked with defect types: {[*ReportPortal.defect_types.keys()]}
-        '''
+    '''
     parser.addoption("--only-failed", const='all', nargs='?', help=help_text)
     help_text = '''
         Collects only skipped tests in Report Portal to run only skipped tests.
 
         Usage: --only-skipped
-        '''
+    '''
     parser.addoption("--only-skipped", action='store_true', default=False, help=help_text)
     help_text = '''
-            Collects user tests from report portal.
+        Collects user tests from report portal.
 
-            Usage: --user [value]
+        Usage: --user [value]
 
-            Value: [ Report_portal_username ]
-
-            Defect_types:
-                Collect tests of specific user from Report Portal
-        '''
+        Value: [ Report_portal_username ]
+    '''
```

Comment:
This change is not related to the upgrade rerun improvement. This is just a nitpick. Just FYI to avoid the confusion.